bio_phil/bio_phil/settings.py

```diff
@@ -14,7 +14,8 @@
 
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
```

Comment:
change the MEDIA_ROOT to point to the static folder you can do this by concatenating "webapp/static/" so itll look something like this:

`MEDIA_ROOT = os.path.join(BASE_DIR+"webapp/static/, 'media)`